src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/CohostSemanticTokensRangeEndpointTest.cs

```diff
@@ -97,8 +98,8 @@ private async Task VerifySemanticTokensAsync(string input, bool colorBackground,
         var legend = TestRazorSemanticTokensLegendService.Instance;
 
         // We need to manually initialize the OOP service so we can get semantic token info later
-        var legendService = OOPExportProvider.GetExportedValue<RemoteSemanticTokensLegendService>();
-        legendService.SetLegend(legend.TokenTypes.All, legend.TokenModifiers.All);
+        var legendService = OOPExportProvider.GetExportedValue<ISemanticTokensLegendService>();
```

Comment:
Unused?

Reply:
Yep, got confused between legendService and legend. Removed the unused legendService.